diff --git a/node_modules/react-native-math-view/android/build.gradle b/node_modules/react-native-math-view/android/build.gradle
index e87b89e..570d473 100644
--- a/node_modules/react-native-math-view/android/build.gradle
+++ b/node_modules/react-native-math-view/android/build.gradle
@@ -5,6 +5,7 @@ def safeExtGet(prop, fallback) {
 }
 
 android {
+    namespace "com.autodidact"
     compileSdkVersion safeExtGet("compileSdkVersion", 28)
     //buildToolsVersion safeExtGet("minSdkVersion", "23.0.1")
 
@@ -31,7 +32,7 @@ repositories {
 buildscript {
     repositories {
         maven { url "https://maven.google.com" }
-        jcenter()
+        mavenCentral()
         google()
         maven {
             url 'https://maven.google.com/'
@@ -51,7 +52,7 @@ allprojects {
     repositories {
         mavenLocal()
         maven { url "https://maven.google.com" }
-        jcenter()
+        mavenCentral()
         google()
         maven {
             // All of React Native (JS, Obj-C sources, Android binaries) is installed from npm
